schema/modules/grpcModule/GrpcModule.go

```diff
@@ -33,20 +39,21 @@ func (m *GrpcModule) GetKey() common.ModuleKey {
 	return grpcModuleKey
 }
 
-//func (m *GrpcModule) Close() {
-//	//close router
-//}
+func (m *GrpcModule) Close() {
+	m.GrpcRouter.Close()
```

Comment:
We need GrpcRouter to call the close function instead of call GrpcRouter.close function outside this method.
It looks strange